sdk/translation/azure-ai-translation-text/azure/ai/translation/text/__init__.py

```diff
@@ -6,11 +6,14 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._patch import TextTranslationClient, TranslatorCredential, TranslatorAADCredential
+from ._patch import TextTranslationClient
 from ._version import VERSION
 
 __version__ = VERSION
 
+
+from ._patch import TranslatorCredential
+from ._patch import TranslatorAADCredential
```

Comment:
With the rebranding from AAD --> Entra ID, should we rename this credential?

Reply:
Done